### PR TITLE
Create LoginService and LoginController

### PR DESCRIPTION
**Pull request by C-Tate**
Hopefully this does everything it needs to

### CODE REVIEW

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/services/LoginService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class LoginService {
+    private UserService userService;
+    
+    public boolean login(String username, String password) {
+
+        User user = userService.findUserByEmail(username);
+
+        if (username.isEmpty()) {
+            throw new IllegalArgumentException("Email cannot be empty.");
+        }
```
Comment (Blacklotus89898) [inline McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/services/LoginService.java:22]: use else if imo

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/dto/LoginRequestDTO.java
Comment (Blacklotus89898) [inline McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/dto/LoginRequestDTO.java:1]: either remove the commented code or uncomment it, no need to save it for later

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/controllers/LoginController.java
Comment (Blacklotus89898) [inline McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/controllers/LoginController.java:1]: I assume loginService exist, LGTM aside from the comments

**Review by Blacklotus89898 — CHANGES_REQUESTED** [review]

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/controllers/LoginController.java
Comment (C-Tate) [reply McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/controllers/LoginController.java:1]: Yeah loginService was added in this same commit, u should be able to see it

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/services/LoginService.java (hunk 1 above)
Comment (C-Tate) [reply McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/services/LoginService.java:22]: got it

File: McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/dto/LoginRequestDTO.java
Comment (C-Tate) [reply McGillMart/McGillMart/src/main/java/com/mcgillmart/McGillMart/dto/LoginRequestDTO.java:1]: ok

**Review by Blacklotus89898 — APPROVED** [review]
Lgtm

**Pull request merged**